script.js

```diff
@@ -0,0 +1,84 @@
+const calculateBtn = document.querySelector('.calculate-btn');
+calculateBtn.addEventListener("click", calculateBtnClick);
```

Comment:
karox eq grel aveli karc aranc nor class stexcelu uxxaki button-in taq funkcia onclick anvanumov vor@ kkrcati dzer kod@.

Reply:
the version you specified is good from the point of view of short code, but since I prefer to separate the business logic from the layout and moreover be able to access the elements in the script in order to directly perform all manipulations with them in the future, I chose this approach.